README.md

```diff
@@ -1,5 +1,7 @@
 # sso-wall-of-shame
 
+This is a fork of the original but now unmaintained [robchahin/sso-wall-of-shame](https://github.com/robchahin/sso-wall-of-shame) which was published at https://sso.tax
```

Comment:
I feel like we should flesh this out a bit more to explain why a fork was necessary.  In particular, that there this isn't for a change in direction, but rather an effort to keep the work going.  Maybe something like:

```suggestion
This is a fork of the original but now unmaintained [robchahin/sso-wall-of-shame](https://github.com/robchahin/sso-wall-of-shame) which was published at https://sso.tax. The growing backlog of requests in that repository was a clear indicator that the good work started by Rob is still needed. While we would've preferred to work with Rob on his existing effort and [help shoulder the load](https://github.com/robchahin/sso-wall-of-shame/issues/163), it would seem that life has pulled him in other directions enough that he wasn't able to respond to us.  Hopefully, at some point in the future our efforts can be merged again.
```